content/id/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -0,0 +1,1202 @@
+web-0     1/1       Running   0         19s
+web-1     0/1       Pending   0         0s
+web-1     0/1       Pending   0         0s
+web-1     0/1       ContainerCreating   0         0s
+web-1     1/1       Running   0         18s
+```
+Perhatikan Pod `web-1` tidak dijalankan hingga `web-0` berganti status menjadi _Running_
+(lihat [Fase Pod](/id/docs/concepts/workloads/pods/pod-lifecycle/#fase-pod)) dan _Ready_
+(lihat `type` di [Kondisi Pod](/id/docs/concepts/workloads/pods/pod-lifecycle/#kondisi-pod)).
+
+## Pod pada StatefulSet
+
+Pod pada StatefulSet memiliki satu indeks urutan unik dan satu identitas jaringan yang tetap.
+
+### Memeriksa Indeks Urutan Pod
+
+Dapatkan Pod dari StatefulSet:
+
+```shell
+kubectl get pods -l app=nginx
+```
+```
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     1/1       Running   0          1m
+web-1     1/1       Running   0          1m
+```
+
+Sebagaimana telah disebutkan di dalam konsep 
+[StatefulSet](/id/docs/concepts/workloads/controllers/statefulset/),
+Pod pada StatefulSet memiliki suatu identitas yang melekat (_sticky_) dan unik.
+Identitas ini didasarkan pada sebuah indeks urutan yang unik yang di tetapkan 
+ke masing-masing Pod oleh {{< glossary_tooltip term_id="controller" text="pengontrol">}} StatefulSet.
+Nama Pod memiliki format `<nama statefulset>-<indeks urutan>`.
+Karena StatefulSet `web` memiliki dua replika, maka ada dua Pod yang tercipta, `web-0` dan `web-1`.
+
+### Menggunakan Identitas Jaringan yang Tetap
+
+Setiap Pod memiliki nama hos yang tetep berdasarkan indeks urutannya. Gunakan perintah
+[`kubectl exec`](/docs/reference/generated/kubectl/kubectl-commands/#exec) untuk menjalankan
+perintah `hostname` di tiap Pod:
+
+```shell
+for i in 0 1; do kubectl exec "web-$i" -- sh -c 'hostname'; done
+```
+```
+web-0
+web-1
+```
+
+Gunakan perintah [`kubectl run`](/docs/reference/generated/kubectl/kubectl-commands/#run) untuk
+menjalankan sebuah Container yang menyediakan perintah `nslookup` dari paket `dnsutils`.
+Dengan menjalankan perintah `nslookup` dengan nama hos dari Pod, kamu dapat memeriksa alamat
+DNS mereka di dalam klaster:
+
+```shell
+kubectl run -i --tty --image busybox:1.28 dns-test --restart=Never --rm
+```
+perintah itu akan memulai sebuah _shell_ baru. Pada _shell_ tersebut, jalankan:
+```shell
+# Jalankan ini di dalam shell Container dns-test
+nslookup web-0.nginx
+```
+Keluarannya akan seperti:
+```
+Server:    10.0.0.10
+Address 1: 10.0.0.10 kube-dns.kube-system.svc.cluster.local
+
+Name:      web-0.nginx
+Address 1: 10.244.1.6
+
+nslookup web-1.nginx
+Server:    10.0.0.10
+Address 1: 10.0.0.10 kube-dns.kube-system.svc.cluster.local
+
+Name:      web-1.nginx
+Address 1: 10.244.2.6
+```
+
+(dan selanjutnya keluarlah dari _shell_ Container dengan menjalankan: `exit`)
+
+CNAME dari _headless service_ mengarah ke SRV _record_ (satu untuk tiap Pod yang
+Running dan Ready). SRC _record_ mengarah ke entri A _record_ yang memuat
```

Comment:
nit: this might need italic
```suggestion
_Running_ dan _Ready_). SRC _record_ mengarah ke entri A _record_ yang memuat
```

Reply:
also SRC is a typo, should be SRV
```suggestion
Running dan Ready). SRV _record_ mengarah ke entri A _record_ yang memuat
```

Reply:
Running and Ready is the status name (also Terminating), I do write them as _Running_ and _Ready_ in line 125 & 126 (first mention) which is the same as the English version.

Reply:
But if for our translation it's preferable for them to be in _italic_ maybe I will update it later in separate PR since this already merged. Do we have some guidelines for this @girikuncoro?

Reply:
@habibrosyad  actually you are right. we are generally following the upstream english styleguide, it's mentioned [here](https://kubernetes.io/docs/contribute/style/style-guide/#use-normal-style-for-string-and-integer-field-values) that we need to use normal style for string field values. This means Running and Ready should be written in normal style with capital.

we are planning to standardized all pages that we have translated already in the next milestone, with the style guide. Do suggest if you have some ideas and findings.